components/ShadowElevations/examples/ShadowElevationsPointsLabel.h

```diff
@@ -0,0 +1,25 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialShadowElevations.h"
+
+@interface ShadowElevationsPointsLabel : UILabel
+
+- (void)setElevation:(MDCShadowElevation)points;
```

Comment:
If this were a property it would improve the readability of the swift examples because you could use a setter.